alexa/response/standardcard.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Alexa\Response;
+
+/**
+ * StandardCard Class
+ * Handles creating and returning a Standard app card.
+ * Standard app cards can contain an image, which differentiates them from Simple cards.
+ */
+class StandardCard {
+	/**
+	 * Type of card
+	 * @access public
+	 * @var string
+	 */
+	public $type = 'Standard';
+	/**
+	 * Title of card
+	 * @access public
+	 * @var string
+	 */
+	public $title = '';
+	/**
+	 * Content of card
+	 * @access public
+	 * @var string
+	 */
+	public $content = '';
+	/**
+	 * Holds the card images
+	 * @access public
+	 * @var null
+	 */
+	public $image = null;
+
+	/**
+	 * Constructor. Sets up the Standard card.
+	 * @param string $title title of card
+	 * @param string $content content of card
+	 * @param int $image ID of image for card
+	 * @access public
+	 */
+	public function __construct( $title, $content, $image ) {
+		$this->title = $title;
+		$this->content = $content;
+		if ( false === ( $images = get_transient( 'voicewp_card_image_' . $image ) ) ) {
+			$images = array(
+				'smallImageUrl' => wp_get_attachment_image_src( absint( $image ), 'alexa-small' )[0],
```

Comment:
Not really code review feedback, but we added a couple thumbnail sizes for the Apple News plugins and users that still actually generate physical thumbnails and don't use photon complained about the extra volume of images. Probably not a concern for VIP but for the wider community of small sizes. Just FYI